uc.c

```diff
@@ -2434,6 +2452,10 @@ uc_err uc_context_restore(uc_engine *uc, uc_context *context)
 UNICORN_EXPORT
 uc_err uc_context_free(uc_context *context)
 {
+    if (context->fv) {
+        free(context->fv->ranges);
+    }
+    g_free(context->fv);
```

Comment:
I think this shall move to the `if (context->fv)` branch.

Reply:
g_free() and free() with a NULL is a noop so it's not necesary. I can move it if you prefer it this way better.

Reply:
Yes, please do so because it looks more straightforward.